Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build the manager binary
+FROM golang:1.13 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+# FROM gcr.io/distroless/static:nonroot
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+WORKDIR /
+COPY --from=builder /workspace/manager .
+USER nonroot:nonroot
+
+ENTRYPOINT ["/manager"]
```

Comment:
this looks wrong

Reply:
This Docker file is not used. Is automatically generated by the operator-sdk.
The common convention I found following other migrations and their documents is to let the skeleton AS IS and use only what we need. I can remove it, of course. Our current project does not remove it, it just remove all the code inside and points to the right direction. Maybe we should do the same.

Reply:
I'm fine removing the boilerplate we don't need in a later commit or even in a later series; any way we should not keep around unused stuff, it's just confusing.

Reply:
Will remove not-needed files/Dirs. At tomorrow's meeting I will bring this up. It seems the standard way to generate operators is to generate the layout, then keep stuff around as templates so all operators will look the same. We can make them empty files, of course. I will be OK with any decision we take. (We'll talk tomorrow)

Reply:
Used another approach for the  docker file - added:
# Dummy file for operator-sdk for detecting project structure
# Please use openshift-ci/Dockerfile.deploy 
like we had before